frontend/src/utils/serializeFilterParams.js

```diff
@@ -47,7 +48,7 @@ export default function serializeFilterParams(filters, descriptions) {
           else
             key += "__icontains"
         } else {
-          key += "__icontains"
+            key += "__icontains"
```

Comment:
I think the indentation was actually correct before. Front end has 2 space indentation, back end 4 spaces.

Reply:
Shouldn't line 51 should be indented since it is inside an else statement?

Reply:
yeah but the else should be aligned where the "}" is.

Reply:
Still too much indentation there...